src/lib.rs

```diff
@@ -312,6 +315,23 @@ impl DecodeReaderBytesBuilder {
         self.bom_override = yes;
         self
     }
+
+    /// Enable BOM sniffing
+    ///
+    /// When this is enabled and an explicit encoding is not set, the decoder will try to detect
+    /// the encoding with BOM.
+    ///
+    /// When this is disabled and an explicit encoding is not set, the decoder will treat the input
+    /// as raw bytes.
```

Comment:
I would add, "The bytes will be passed through unchanged, including any BOM that may be present."

Reply:
Done!